admin/class-yoast-form.php

```diff
@@ -317,10 +317,9 @@ public function light_switch( $var, $label, $buttons = array(), $reverse = true,
 		$help_class = ! empty( $help ) ? ' switch-container__has-help' : '';
 
 		echo "<div class='switch-container$help_class'>",
-		"<span class='switch-light-visual-label'>{$label}</span>" . $help,
+		"<span class='switch-light-visual-label' id='{$id}'>{$label}</span>" . $help,
```

Comment:
When touching this, please use the double quotes on HMTL attributes for consistency.